### PR TITLE
feature/delete response button

### PR DESCRIPTION
**Pull request by GV14982**
- 💄 Adds delete button to Responses UI - ⬆️ Updates 5.0.0-alpha1 to 5.0.0-alpha3 - 💄 Adds some super basic styles to index.scss for delete button - 💄 Adds .response-preview to .card for each response
- ✨ Added delete logic/modal

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,6 +14,9 @@ import { Error404 } from './views/Error404';
 import { Response } from './views/Response';
 import { Responses } from './views/Responses';
 
+if (process.env.NODE_ENV === 'production')
+  axios.defaults.baseURL = process.env.REACT_APP_BASE_URL;
+
```
[the remark this answers is not in the document]
Comment (GV14982) [reply src/App.jsx:19]: API is hosted on Linode at api.backtalk.io
UI is hosted on Netlify at backtalk.io

File: src/components/Modal.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  return show
+    ? createPortal(
+        <div
+          style={{
+            position: 'absolute',
+            width: '100vw',
+            height: '100vh',
+            top: '0px',
+            left: '0px',
+          }}
```
[the remark this answers is not in the document]
Comment (GV14982) [reply src/components/Modal.jsx:14]: Yeah I know I was just lazy haha.

File: src/views/Response.jsx [hunk 3]
```diff
@@ -236,7 +236,7 @@ export const Response = ({ location }) => {
                 {!surveyEnd(survey?.current, cursor) &&
                   survey?.current?.questions && (
                     <input
-                      autoFocus="true"
+                      autoFocus={true}
```
[the remark this answers is not in the document]
Comment (GV14982) [reply src/views/Response.jsx:239]: Yeah, anytime you are using actual JSX, you need to surround it with curly braces.

**Comment by seanmetzgar** [comment]
You gots some merge conflicts... you want me to tackle those?

**Review by seanmetzgar — APPROVED** [review]

**Pull request merged**